gtsfm/scene_optimizer.py

```diff
@@ -172,29 +176,11 @@ def create_computation_graph(
                         keypoints_graph_list[i1],
                         keypoints_graph_list[i2],
                         v_corr_idxs,
-                        two_view_report=two_view_report,
+                        two_view_report=two_view_reports[PRE_BA_REPORT_TAG],
                         file_path=os.path.join(PLOT_CORRESPONDENCE_PATH, f"{i1}_{i2}.jpg"),
                     )
                 )
 
-        # Persist all front-end metrics and their summaries.
-        auxiliary_graph_list.append(
```

Comment:
why is this removed?

Reply:
Its done later for all the different reports.